Octokit.Tests.Integration/Clients/PullRequestsClientTests.cs

```diff
@@ -59,6 +59,7 @@ public PullRequestsClientTests()
 
         Assert.Equal(1, pullRequests.Count);
         Assert.Equal(result.Title, pullRequests[0].Title);
+        Assert.NotNull(pullRequests[0].Id);
```

Comment:
This `int` field can never be `null` so I'm thinking we will need to assert something else here (eg `Assert.True(pullRequests[0].Id > 0)`